indico/modules/vc/templates/manage_event.html

```diff
@@ -36,8 +36,10 @@
                                                 <a class="i-button icon-loop js-vcroom-refresh"
                                                    title="{% trans %}Check videoconference room status{% endtrans %}"
                                                    data-href="{{ url_for('.manage_vc_rooms_refresh', event_vc_room)}}"></a>
-                                                <a href="{{ url_for('.manage_vc_rooms_modify', event_vc_room) }}"
-                                                   class="i-button icon-edit js-vcroom-edit"
+                                                <a href=# class="i-button icon-edit js-vcroom-edit"
```

Comment:
we always quote attribute values, i.e. `href="#"`. also, move the title attribute in the line below to have the data attributes at the end